src/quic/node_quic_util-inl.h

```diff
@@ -42,6 +42,12 @@ size_t QuicCID::Hash::operator()(const QuicCID& token) const {
   return hash;
 }
 
+QuicCID& QuicCID::operator=(const QuicCID& cid) {
+  if (this == &cid) return *this;
+  this->~QuicCID();
+  return *new(this) QuicCID(std::move(cid));
```

Comment:
Something is off about this – the `std::move` here won’t do anything to a const reference, so either you meant to implement a move operator rather than a copy operator, or the `std::move` can be dropped?

Reply:
Actually, I don't think we're actually using this at all so likely best to just remove it.